src/moov/trak/mdia/minf/stbl/stsd/mod.rs

```diff
@@ -88,6 +92,18 @@ pub enum Codec {
     // EAC-3 audio
     Eac3(Eac3),
 
+    // Uncompressed audio
```

Comment:
Which standard do these come from? Couldn't see them in 23003-5.

Reply:
Only `ipcm` and `fpcm` are 23003-5.

`sowt` /  `twos` and `in24` / `in32` / `fl32` / `fl64` are Quicktime (QTFF-2001). `s16l` seems to be something that VLC produced at some point. `lpcm` comes from Quicktime again (https://developer.apple.com/documentation/quicktime-file-format/sound_sample_description_version_2).